homeassistant/components/homeassistant/strings.json

```diff
@@ -71,6 +71,10 @@
           }
         }
       }
+    },
+    "ssl_configured_without_external_url": {
+      "title": "SSL is configured without an external URL",
+      "description": "SSL is configured, but no external URL is set. Please set `external_url` in your configuration."
```

Comment:
You could add some fluff and add instructions on how and where to solve this issue + a link to that location in the settings (take the users right to it)

Suggestion for text:

---

Home Assistant detected that SSL has been set up on your instance, however, no custom external internet URL has been set.

This may result in unexpected behavior. Text-to-speech may fail, and integrations may not be able to connect back to your instance correctly.

To address this issue, go to **Settings** > **System** > **Network**; under the "Home Assistant URL" section, configure your new "Internet" and "Local network" addresses that match your new SSL configuration.